frontend/.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "conventionalCommits.scopes": [
```

Comment:
using the extension `vivaxy.vscode-conventional-commits`, the `scopes` are shared between devs